### DISCUSSION TITLE
Curated third-party flatpaks by default

### DISCUSSION THREAD

**Discussion by Askkegra** [elementary/triage#385] [Ideas]
I imagine the idea of enabling FlatHub by default is a recurring question/discussion, as it is often a common talking point whenever eOS is brought up online. But the hesitation to do so is valid as well.

I do not know exactly the feasibility of this idea, but I'd like to make the suggestion to add a curated set of popular third-party apps to be available as flatpaks on the App Center by default. You can narrow it down to popular open-source software with a good track record, select for apps that complement current native offerings, list them below native apps, provide users with warnings as needed, etc.

**Comment by lenemter** [comment]
Flathub is now enabled by default in OS 8. I'll close the discussion.
